src/v2_5/mod.rs

```diff
@@ -30,6 +30,8 @@ mod source;
 mod supply_chain;
 mod user;
 mod video;
+mod data;
+mod segment;
```

Comment:
you need to add a public usage of your mods to use their structs outside of a crate

```
pub use self::data::*;
pub use self::segment::*;
```

Reply:
fixed